### ISSUE TITLE
Topology is displayed in a wrong way

### ISSUE THREAD

**Issue by Guenfo** [martimy/flowmanager#5]
Hello sir,
I hope you can help me I created a mininet topology using the following command: 
`sudo mn --controller=remote,ip=127.0.0.1 --mac --switch=ovsk,protocols=OpenFlow13 --topo=linear,4`
and after starting a simple switch script in ryu controller I get a wrong topology and the real topology is that every host is connected to one switch :
![mininet-topo](https://user-images.githubusercontent.com/65870496/89107276-21def500-d430-11ea-9083-ef27325c5017.png)

**Comment by martimy** [comment]
To display the topology correctly you have to use the --observe-links options with ryu-manager.
$ ryu-manager --observe-links ~/flowmanager/flowmanager.py <your app>

**Comment by Guenfo** [comment]
> To display the topology correctly you have to use the --observe-links options with ryu-manager.
> $ ryu-manager --observe-links ~/flowmanager/flowmanager.py

thank you sir for replying I already did that and still it shows wrong topology

**Comment by martimy** [comment]
The option (--observe-links) adds an entry to Table 0 with the highest priority (as shown below). Make sure this entry is there and not deleted by your application.

![image](https://user-images.githubusercontent.com/4184429/90457545-3da1f680-e0d2-11ea-8ca2-a4d7038dc85c.png)

If you still not getting the topology correct, 1) review your topology and ensure it is correct, and 2) use one of the Ryu example applications before you try your own (e.g. simple_switch_13.py).

I tried your topology above and I got the following 


![image](https://user-images.githubusercontent.com/4184429/90458410-b86c1100-e0d4-11ea-85cb-09f3be53e638.png)


$ sudo mn --controller=remote,ip=127.0.0.1 --mac --switch=ovsk,protocols=OpenFlow13 --topo=linear,4
$ ryu-manager --observe-links flowmanager/flowmanager.py ryu.app.simple_switch_13
